src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let accamulate = 0;
```

Comment:
I think you`ve meant here accumulate (or accumulator?)

Reply:
> I think you`ve meant here accumulate (or accumulator?)

Exactly)